### PR TITLE
Add Support for EXPRESS_PORT Variable

### PR DESCRIPTION
**Pull request by todd**
As discussed in #728, this PR adds support for an `EXPRESS_PORT` variable to set the port for Express to listen on while also retaining support for the current `PORT` variable. `EXPRESS_PORT` will be favored over `PORT`. My only outstanding question is whether we should also add `EXPRESS_BIND_ADDRESS` - this hasn't been an issue I've run into, but it might be nice for the sake of consistency.

cc: @technicalpickles

### CODE REVIEW

File: src/robot.coffee [hunk 1]
```diff
@@ -269,6 +269,8 @@ class Robot
     user    = process.env.EXPRESS_USER
     pass    = process.env.EXPRESS_PASSWORD
     stat    = process.env.EXPRESS_STATIC
+    port    = process.env.EXPRESS_PORT or process.env.PORT or 8080
+    address = process.env.BIND_ADDRESS or '0.0.0.0'
```
Comment (michaelansel) [inline src/robot.coffee]: I don't see any reason not to do the same namespacing with BIND_ADDRESS. Leaving it as the lone non-namespaced variable for Express just seems strange.

Comment (todd) [inline src/robot.coffee]: Yeah, that was more or less my thinking as well, but I'm waiting for official word from a maintainer that that's what they want.

Comment (technicalpickles) [inline src/robot.coffee]: `BIND_ADDRESS` was added in https://github.com/github/hubot/pull/609 . I'd be fine namespacing it, as long as the old BIND_ADDRESS is used as well.

**Comment by technicalpickles** [comment]
I think this should be good to merge, once EXPRESS_BIND_ADDRESS is supported as well.

**Comment by todd** [comment]
@technicalpickles I went ahead and added that. Should be good to go.

**Comment by todd** [comment]
@technicalpickles Any chance we can get this merged?

**Comment by todd** [comment]
:heart:

**Pull request merged**